eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -42,9 +42,19 @@ function Check-ApiReviewStatus($packageName, $packageVersion, $language, $url, $
     {
       Write-Host "API Review is approved for package $($packageName)"
     }
+    else if ($response.StatusCode -eq '202')
+    {
+      Write-Host "Package name $($packageName) is not yet approved by an SDK API approver. Package name must be approved to release a preview version if $($packageName) was never released as GA version."
```

Comment:
```suggestion
      Write-Host "Package name $($packageName) is not yet approved by an SDK API approver. Package name must be approved to release a beta version if $($packageName) was never released a stable version."
```

Reply:
minor but we have been slowing trying to refer to our package releases as beta or stable as opposed to preview and GA.